src/test/java/org/dita/dost/ITPreprocess2.java

```diff
@@ -14,7 +14,7 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-public class ITPreprocess2 extends AbstractIntegrationTest implements ITPreprocessBase {
+public class ITPreprocess2 extends AbstractIntegrationTest implements ITPreprocessBase, ITContentUplevels {
```

Comment:
Because `ITFormatHtml5` already uses preprocess2, also running the uplevels tests here is duplicated tests. Thus IMO this should be removed to make the integration tests faster.